website/server/controllers/api-v3/chat.js

```diff
@@ -412,9 +412,9 @@ api.seenChat = {
  * @apiName DeleteChat
  * @apiGroup Chat
  *
- * @apiParam {String} previousMsg Query parameter - The last message fetched by the client so that the whole chat will be returned only if new messages have been posted in the meantime
- * @apiParam {UUID} groupId The group _id ('party' for the user party and 'habitrpg' for tavern are accepted)
- * @apiParam {UUID} chatId The chat message id
+ * @apiParam (Query) {String} previousMsg The last message fetched by the client so that the whole chat will be returned only if new messages have been posted in the meantime
```

Comment:
I'm pretty sure this needs to be a UUID like this: https://github.com/HabitRPG/habitica/pull/8277/files#diff-ade116c537945ae89e2f5f059d4d577fR80